gsad/src/gsad_http_handler.c

```diff
@@ -308,7 +308,7 @@ handle_validate (http_connection_t *connection, const char * method,
   if (&url[0] == NULL)
     {
       send_response (connection, BAD_REQUEST_PAGE, MHD_HTTP_NOT_ACCEPTABLE,
-                     NULL, GSAD_CONTENT_TYPE_TEXT_HTML, NULL, 0);
+                     NULL, GSAD_CONTENT_TYPE_TEXT_HTML, NULL, 0, 0);
```

Comment:
e.g. this is nearly unreadable without knowing the function signature of send_response. It won't get better with another int param.